src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/SubmissionManagerImpl.java

```diff
@@ -19,50 +11,79 @@
 import org.restlet.representation.Representation;
 import org.restlet.resource.ClientResource;
 
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
 /**
  * Concret implementation of {@link SubmissionManager}.
  */
 public class SubmissionManagerImpl implements SubmissionManager {
 
     private static final Logger LOGGER = Logger.getLogger(SubmissionManagerImpl.class);
 
-    // TODO: Replace static string by config txt file
-    private static final String REMOTE_INSTANCE_URL = "";
+    private static final String SAPS_NEIGHBORS_URLS = "saps_neighbors_urls";
+    private static final String PROCESSED_TASKS_URN = "/archivedTask";
```

Comment:
out of curiosity ... is it URN or URL?

Reply:
It's URN, because it's just the resource name